pages/api/user.ts

```diff
@@ -1,11 +1,11 @@
 import { superTokensNextWrapper } from 'supertokens-node/nextjs'
 import { verifySession } from 'supertokens-node/recipe/session/framework/express'
-import supertokens from 'supertokens-node'
+import { SessionRequest } from 'supertokens-node/framework/express'; import supertokens from 'supertokens-node'
 import * as SuperTokensConfig from '../../config/backendConfig'
 
 supertokens.init(SuperTokensConfig.backendConfig())
 
-export default async function user(req, res) {
+export default async function user (req: SessionRequest, res: any): Promise<any> {
```

Comment:
Couldn't these `any` be more specifically typed?

Reply:
fixed, thanks!

Reply:
improved it a bit now